lib/src/content/components/main_content/wolt_modal_sheet_top_bar_title.dart

```diff
@@ -37,12 +37,12 @@ class _WoltModalSheetTopBarTitleState extends State<WoltModalSheetTopBarTitle> {
   }
 
   void _extractPageTitleText() {
-    final isPageTitleTextWidget = widget.page.pageTitle is Text;
+    // final isPageTitleTextWidget = widget.page.pageTitle is Text;
```

Comment:
Are we sure this is unnecessary? This is checking if the current widget is text widget. If not, it is checking all the descendents of the current widget.

Reply:
revert it, there is a if block that is empty it seems